src/plugins/opensearch_dashboards_react/public/table_list_view/components/create_button_menu.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React, { useState } from 'react';
+import {
+  EuiButton,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFlexItem,
+  EuiPopover,
+} from '@elastic/eui';
+import type { DashboardCreator, DashboardCreatorFn, DashboardCreators } from '../../context/types';
+
+interface CreateButtonProps {
+  // createItem: () => {};
+  dashboardItemCreatorClickHandler: (creatorFn: DashboardCreatorFn) => (event: any) => void;
+  dashboardItemCreators: () => DashboardCreators;
+}
+
+const CreateButton = (props: CreateButtonProps) => {
+  const [isPopoverOpen, setPopover] = useState(false);
+
+  const onButtonClick = () => {
+    setPopover(!isPopoverOpen);
+  };
+
+  const closePopover = () => {
+    setPopover(false);
+  };
+
+  const getItems = (creators: DashboardCreators) => {
+    return creators.map((creator: DashboardCreator) => (
+      <EuiContextMenuItem
+        key={creator.id}
+        onClick={props.dashboardItemCreatorClickHandler(creator.creatorFn)}
+      >
+        {creator.defaultText}
+      </EuiContextMenuItem>
+    ));
+  };
+
+  const forceSingleButtonOnly = false;
+
+  const creators: DashboardCreators = props.dashboardItemCreators();
+
+  const renderMenu = () => {
+    if (creators.length === 0) {
+      return null;
```

Comment:
Will this case ever be true?

Reply:
Outdated code.  Moved to plugins/dashboard